DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: s3mpi
 Title: R message passing interface using S3 storage
 Description: Easily pass objects like lists or dataframes between consoles
-Version: 0.2.2
+Version: 0.2.3
```

Comment:
Maybe 0.2.2.1?  Don't know how @robertzk wants to do versioning.  Also don't know if he wants bugfixes in `NEWS.md`.